lib/matplotlib/__init__.py

```diff
@@ -1476,7 +1476,6 @@ def _jupyter_nbextension_paths():
 default_test_modules = [
     'matplotlib.tests.test_png',
     'matplotlib.tests.test_units',
-    'matplotlib.tests.test_widgets',
```

Comment:
Yes, that's why I pass only the top-level modules; but we still need to pass something because this works on the installed version and we need to hit both `matplotlib` and `mpl_toolkits`. I wouldn't be against deprecating `matplotlib.test()` and the top-level `tests.py`.